data/risingwave/models/marpower_energy_docs.md

```diff
@@ -0,0 +1,64 @@
+<!-- Electrical energy metrics -->
+{% docs active_power_on_phase %}
+The active power (W) measured on phase
+{% enddocs %}
+{% docs power_factor_on_phase %}
+The power factor measured on phase
+{% enddocs %}
+{% docs rms_current_on_phase %}
+The RMS current (A) measured on phase
+{% enddocs %}
+{% docs rms_voltage_on_phase %}
+The RMS voltage (V) measured on phase
+{% enddocs %}
+{% docs total_active_power_cons %}
+The electrical energy (W) currently being consumed.
+{% enddocs %}
+{% docs total_active_power_prod %}
+The electrical energy (W) currently being produced.
+{% enddocs %}
+{% docs active_power %}
+The electrical energy (W) currently being consumed or produced.
+{% enddocs %}
+{% docs power_factor %}
+The power factor of the measured power.
+{% enddocs %}
+{% docs voltage_at_side %}
+The measured voltage (V) at side
+{% enddocs %}
+{% docs current_at_side %}
+The measured current (A) at side
+{% enddocs %}
+{% docs stored_energy %}
+The electrical energy (Wh) stored in the battery.
```

Comment:
I don't think Wh is something we should expect (or want) Marpower to supply. Ideally I'd have Joules, but otherwise kWh makes more sense